config/config.js.sample

```diff
@@ -3,97 +3,97 @@
  * By Michael Teeuw http://michaelteeuw.nl
  * MIT Licensed.
  *
- * For more information how you can configurate this file
+ * For more information how you can configuration this file
  * See https://github.com/MichMich/MagicMirror#configuration
  *
  */
 
 var config = {
-	address: "localhost", // Address to listen on, can be:
-	                      // - "localhost", "127.0.0.1", "::1" to listen on loopback interface
-	                      // - another specific IPv4/6 to listen on a specific interface
-	                      // - "0.0.0.0", "::" to listen on any interface
-	                      // Default, when address config is left out or empty, is "localhost"
```

Comment:
It can also be empty. Why remove this?

Reply:
Figured left out and empty more or less meant the same thing. I'll revert the change. Thanks